hosted_control_planes/hcp-disconnected/hcp-deploy-dc-virt.adoc

```diff
@@ -0,0 +1,70 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="hcp-deploy-dc-virt"]
+include::_attributes/common-attributes.adoc[]
+= Deploying {hcp} on {VirtProductName} in a disconnected environment
+:context: hcp-deploy-dc-virt
+
+toc::[]
+
+When you deploy {hcp} in a disconnected environment, some of the steps differ depending on the platform you use. The following procedures are specific to deployments on {VirtProductName}.
+
+:FeatureName: {hcp-capital} on {VirtProductName} in a disconnected environment
+include::snippets/technology-preview.adoc[]
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* You have a disconnected {product-title} environment serving as your management cluster.
+* You have an internal registry to mirror images on. For more information, see xref:../../installing/disconnected_install/index.adoc[Disconnected installation mirroring].
+
+include::modules/hcp-dc-image-mirror.adoc[leveloffset=+1]
+include::modules/hcp-dc-apply-objects.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about mirroring nightly or CI versions of {product-title}, see xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plugin].
+
+[id="hcp-dc-mce"]
+== Deploying {mce-short} for a disconnected installation of {hcp}
+
+The {mce} plays a crucial role in deploying clusters across providers. If you do not have {mce-short} installed, review the following documentation to understand the prerequisites and steps to install it:
+
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#mce-intro[About cluster lifecycle with multicluster engine operator]
+* link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#mce-install-intro[Installing and upgrading multicluster engine operator]
+
+include::modules/hcp-agentserviceconfig.adoc[leveloffset=+2]
```

Comment:
This section belongs to 'Deploying hosted control planes on bare metal'.

Reply:
Good catch. I moved it out of the virtualization section and into the bare metal section.